lib/screens/home_screen.dart

```diff
@@ -1,19 +1,32 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/widgets.dart';
+import 'package:hive_flutter/hive_flutter.dart';
+import 'package:the_gay_agenda/services/events.dart';
 import 'package:the_gay_agenda/widgets/month_view.dart';
 
-import 'package:the_gay_agenda/constants/example_events.dart';
-
 class HomeScreen extends StatelessWidget {
   const HomeScreen({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: AppBar(
-        title: const Text("The Gay Agenda"),
-      ),
-      body: MonthView(events: exampleEvents),
-    );
+        appBar: AppBar(
+          title: const Text("The Gay Agenda"),
+        ),
+        body: ValueListenableBuilder(
+          valueListenable: Hive.box<Event>("events").listenable(),
+          builder: (_, Box<Event> box, __) =>
+              MonthView(events: box.values.toList()),
+        ),
```

Comment:
Isn't there any way to make this shorter ? What happens when the database has not loaded already ?